VespaGAS/Managers/OilMixCalculator.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+let MLInLiter: Float = 1000
+
+typealias Oil = Volume
+typealias Gasoline = Volume
+
+struct OilMixCalculator {
+    static func oilFor(gasoline: Gasoline, oilMix: OilMix) -> Oil {
```

Comment:
You are absolutely right! 

In my previous iteration I have created this calculator because it was not clear who was responsible of this logic and there were no models for the Gasoline and the Oil. 
With the introduction of the Oil model it would make sense to have this method on this object, so I could introduce something like: 

`func oilFor(gasoline: Gasoline, oilMix: OilMix) `

or 

`init(gasoline: Gasoline, oilMix: OilMix)`

What do you think?